app/javascript/gobierto_dashboards/webapp/lib/config.js

```diff
@@ -16,3 +16,35 @@ export const assigneesColumns = [
   {field: 'count', translation: I18n.t('gobierto_dashboards.dashboards.contracts.contracts'), format: 'quantity'},
   {field: 'sum', translation: I18n.t('gobierto_dashboards.dashboards.contracts.final_amount'), format: 'currency'},
 ];
+
+export const filtersConfig = [
+  {
+    id: 'dates',
+    title: I18n.t('gobierto_dashboards.dashboards.contracts.dates'),
+    isToggle: true,
+    options: []
+  },
+  {
+    id: 'contract_types',
+    title: I18n.t('gobierto_dashboards.dashboards.contracts.contract_type'),
+    isToggle: true,
+    options: [
+      {id: 0, title: 'Gestión de servicios públicos', counter: 0, isOptionChecked: false},
+      {id: 1, title: 'Obras', counter: 0, isOptionChecked: false},
+      {id: 2, title: 'Servicios', counter: 0, isOptionChecked: false},
+      {id: 3, title: 'Suministros', counter: 0, isOptionChecked: false},
+    ]
+  },
+  {
+    id: 'process_types',
+    title: I18n.t('gobierto_dashboards.dashboards.contracts.process_type'),
+    isToggle: true,
+    options: [
+        {id: 0, title: 'Abierto', counter: 0, isOptionChecked: false},
+        {id: 1, title: 'Abierto simplificado', counter: 0, isOptionChecked: false},
+        {id: 2, title: 'Basado en Acuerdo Marco', counter: 0, isOptionChecked: false},
+        {id: 3, title: 'Negociado con publicidad', counter: 0, isOptionChecked: false},
+        {id: 4, title: 'Negociado sin publicidad', counter: 0, isOptionChecked: false}
+      ]
```

Comment:
Untraslated stuff? Oh lord

Reply:
That's the title coming from the csv. I did it this way so make things simple, but maybe I should generate it automatically based on the csv :\

Reply:
@eltercero yes, I think you should make these lists dynamic from the CSV, there might be more types

Reply:
Changed!